pyhoma/models.py

```diff
@@ -509,6 +619,73 @@ def __init__(
         self.sub_places = [Place(**p) for p in sub_places] if sub_places else []
 
 
+@attr.s(auto_attribs=True, init=False, slots=True, kw_only=True)
+class Feature:
+    name: str
+    source: str
+
+    def __init__(
+        self,
+        *,
+        name: str,
+        source: str,
+        **_: Any,
+    ) -> None:
+        self.name = name
+        self.source = source
```

Comment:
I think it’s safe here to write. I don’t think Somfy will add new fields.

```suggestion
@attr.s(auto_attribs=True, slots=True, kw_only=True)
class Feature:
    name: str
    source: str
```